src/Alert/Alert.scss

```diff
@@ -14,22 +14,29 @@
   position: relative;
   padding: $alert-padding-y $alert-padding-x;
   margin-bottom: $alert-margin-bottom;
-  border: $alert-border-width solid transparent;
```

Comment:
Rather than deleting this: let's set `$alert-border-width` to `0` in Alert/_variables.scss